src/harness/unittests/tsserverProjectSystem.ts

```diff
@@ -321,6 +321,34 @@ namespace ts.projectSystem {
         verifyDiagnostics(actual, []);
     }
 
+    function getPathsForTypesOrModules(base: string, rootPaths: string[], typesOrModules: string[], map: Map<true>, rootDir?: string) {
+        while (1) {
+            forEach(rootPaths, r => {

```

Comment:
Nit: If you're just iterating over an array and not returning anything a `for-of` loop looks nicer.